### DISCUSSION TITLE
Add an "are you sure" popup window when you click block checkmark

### DISCUSSION THREAD

**Discussion by RynTurtle** [Chatterino/chatterino2#5565] [Ideas] [answered]
I just accidentally unfollowed someone by clicking the checkbox when having the viewercard open, losing our followage, I suggest having a popup window with an "are you sure" message would be perfect to prevent any stupid missclicks and ensure that it doesnt happen to anyone else.

**Comment by jupjohn** [comment]
If this change gets implemented and a dialog pops up on block I'd suggest changing the block action from a check box to a button. If we want to indicate state (blocked or not), then that could probably be indicated in the summary text (perhaps next to the username in brackets)

**Comment by Mm2PL** [comment]
What if we disallowed clicking the block checkbox if the target is followed. We can still check that iirc. Usually users don't want to block someone who they follow.

**Comment by LeetHaxor** [comment]
Since this was already added, can an option be added to disable this feature? On by default is fine, I'd just like the option.

**Reply by pajlada** [reply]
> Since this was already added, can an option be added to disable this feature? On by default is fine, I'd just like the option. 

I'd rather not add a setting for this. Do you often block users? Maybe there's another way that you can block users instead

**Reply by LeetHaxor** [reply]
I do. I requested the block list increase which was implemented a while ago. 
I'll just figure out how to remove the feature via fork (or however it's done) if it becomes an issue. No worries.

**Reply by pajlada** [reply]
There's the `/block` command which does not give you a popup - you can add it to the right-click option of a user
![image](https://github.com/user-attachments/assets/0cec55ce-0962-4f37-8590-227bb8de2132)
![image](https://github.com/user-attachments/assets/d817a162-c9bf-44b0-b8b3-65d777a733c8)

**Reply by LeetHaxor** [reply]
Thanks, that'll work perfectly.

**Comment by Felanbird** [comment, marked as answer]
added in d3000ba5975d8823c04f1b223711088cf49388e8
